website/cue/reference/components/transforms/remap.cue

```diff
@@ -44,6 +44,14 @@ components: transforms: "remap": {
 		notices: []
 	}
 
+	_example_vrl_program: #"""
+		. = parse_json!(.message)
+		.new_field = "new value"
+		.status = to_int!(.status)
+		.duration = parse_duration!(.duration, "s")
+		.new_name = del(.old_name)
+		"""#
```

Comment:
Just noticing that the `"""` isn't appearing on the right line in the rendered output.